### ISSUE TITLE
[calendar] - Update break calendar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#522]
```
Token: dd12c1d22a08aff4be5253f5be23c247 (Don't modify this token!)
Version: 14.1.5 (23-04-2013)
PHP: 5.3.3
RCMAIL: 0.9.0
Database: mysql
SERVER: Apache/2.2.3 (CentOS)
----
I.  Issue Description:
Update break calendar, links, div sync block

II. Steps to reproduce the Issue:
1. Update calendar
2. Block sync start, gui get mess
3. Charging("Cargando") doesnt stop, and this breaks toolbar links
```

Original issue reported on code.google.com by `mmartine...@gmail.com` on 24 Apr 2013 at 5:56

Attachments:
- [issue.jpg](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-522/comment-0/issue.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Update: Charging take too much, and then breaks links
```

Original comment by `mmartine...@gmail.com` on 24 Apr 2013 at 6:01

**Comment by GoogleCodeExporter** [comment]
```
Works for me. Please mail a test account where we can reproduce the issue to 
myroundcube [at] mail4us [dot] net. Alternatively reproduce the issue @ 
http://s2.myroundcube.com and mail login credentials of your test account.
```

Original comment by `roland.l...@gmail.com` on 24 Apr 2013 at 8:11
- Changed state: **Reproducing**

**Comment by GoogleCodeExporter** [comment]
```
Bug occurs if default view = month.
```

Original comment by `roland.l...@gmail.com` on 24 Apr 2013 at 9:38
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `roland.l...@gmail.com` on 24 Apr 2013 at 9:38
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v 14.1.16.
```

Original comment by `roland.l...@gmail.com` on 24 Apr 2013 at 10:44
- Changed state: **Fixed**
